essay.tex

```diff
@@ -0,0 +1,43 @@
+\documentclass{scrartcl}
+
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{graphicx}
+
+\title{When do self-organising Agile game development teams need managing?}
+\subtitle{COMP150 -Agile Essay}
+
+\author{Student Number: 1607539}
+
+\begin{document}
+
+\maketitle
+
+\section*{The Scrum Master}
+
+Despite the Agile manifesto \cite {fowler2001agile} itself stating as part of its principles to ``build projects around motivated individuals'', ‘`and to trust them to get the job done'', it still acknowledges the need for a scrum master, not necessarily to act as a manager in the traditional sense, more to ensure adherence to the values and practises of scrum and to remove any impediments to the progress.  The scrum master is responsible for the process used during the software development \cite {larusdottir2016stakeholder}.  Within these subsets of the role are the more traditional management processes: scrum practises may require a level headed methodical approach when requirements are changing late in development, whereas impediments to a sprint may need a more lateral problem solving approach.  Both management and teams jointly manage issues, risks, and solutions \cite {schwaber1997scrum} however in a daily stand up the responsibility falls upon the scrum master.  This pivotal role can be filled by any team member per scrum and as such, consideration must be given when choosing between candidates as to their suitability, not just for responsibility, but also their leadership style and how fitting that is to the sprint tasks at hand.
+
+\section*{Management Styles}
+
+Luckily there already exists a well proven model of managing styles which we can adopt to study the repercussions of each in the context of Agile game development.  Whilst the names and number of terms changes, the extreme values of the scale are usually agreed upon.  From Autocratic as the most dominant, to Laissez-faire which translates to soft touch.  Management styles are distinct from each other in the relative importance of desired goals \cite{fairweather1994goals}.  Leaders and managers are concerned with two aspects of their work.  The first of these aspects we term task accomplishment, organizing activities to perform assigned tasks.  The second aspect we term maintenance of interpersonal relationships, tending to the morale and welfare of the people in the setting \cite {eagly1990gender}.  Each manager's own style is weighted on a gradient towards one of these and it is widely accepted that the optimum management team encapsulates the entire range.  It is important here to differentiate between a management team usually operating over a long period and a single scrum master over a much shorter sprint. 
+
```

Comment:
How well-defended is the claim `Each manager's own style is weighted on a gradient towards one of these and it is widely accepted that the optimum management team encapsulates the entire range.`? Citation needed.